examples/astro-forms-demo/src/components/Appointment.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { type AppointmentProps } from '@wix/headless-forms/react';
+
+export default function Appointment({
```

Comment:
I think that there should be a default implementation for these and not just as example, so if the user did not override, they will get the default implementation.
You would probably need ref for these components, since I am guessing the form renderer would use it for positioning,
Also the code is quite repetitive maybe create some compositions or at least ability to define default class names for labels, input, errors (on the Root maybe or something), so that users would not have to implement 10s of components just for simple forms.